### DISCUSSION TITLE
Pynecone app stuck on installing bun

### DISCUSSION THREAD

**Discussion by Siddharth8800** [reflex-dev/reflex#551] [General]
I am using Pynecone in a Conda environment and the launch gets stuck here
<img width="856" alt="image" src="https://user-images.githubusercontent.com/110777935/219398283-0d8e0538-3b05-4cb2-b429-8537d999e8a8.png">

anyone else facing the same issue ?

**Comment by Lendemor** [comment]
You need bun `v0.5.5`, `v0.5.6` get stuck like it happened to you.

**Reply by Siddharth8800** [reply]
Yeah i figured it out, but still thanks a lot man ☺️

**Comment by gsal** [comment]
Silly question...

I am not much of a web developer (that's why I am here, using a pure-python framework) and, so, I did not know about bun until I installed pynecone.  

I read online that bun is supposed to be a replacement for node; if so, why do we also need to install node before we can use pynecone?

**Reply by Alek99** [reply]
Bun has been a great tool to use because of its speed, but it can run into errors sometimes and can't replace everything node does as of now. So we use bun where we can to speed things up and node is there for some things bun can't do  (yet)

**Comment by Jarred-Sumner** [comment]
Please let us know if this is still happening on Bun v0.5.7 (released last week)

**Reply by Siddharth8800** [reply]
Switched to bun v0.5.6 everything working fine since, haven't tried v0.5.7 again
